extras/audioDrivers/SDL/AudioSDL.cpp

```diff
@@ -119,6 +127,7 @@ void AudioSDL::CleanUp()
     AudioDriver::CleanUp();
 
     Mix_CloseAudio();
+    currentInstance = nullptr;
```

Comment:
should we check if its the same instance, not that we clean up the wrong one?

Reply:
If you have 2 instances active and running here you'll have bigger problems I guess. Mix_CloseAudio() likely cancels everything already so there won't be any calls to the callback and hence this won't be used again